File: Session-3/shapes/Triangle.java
public class Triangle extends Shape {

  private double height;

  public void setHeight(double height) {
    this.height = height;
  }

  public double getHeight() {
    return height;
  }

  @Override
  public double area() {
    double base = getDim();
    return 0.5 * base * height;
  }
}

File: Kotlin/Lap 2.1/src/Triangle.kt
class Triangle(): Shape() {
    private var height: Double=0.0
   constructor(b: Double, h: Double): this(){
       setDim(b)
       height = h
   }
    fun getHeight(): Double{
        return height
    }
    fun setHeight(h: Double){
        height=h
    }

    override fun calcArea(): Double {
        return 0.5 * getDim() * height
    }
}